spec/integration/_dev/deploy/agent/custom-agent.spec.yml

```diff
@@ -13,11 +13,20 @@ spec:
         anyOf:
           - required:
             - hostname
+    volume:
+      type: "null"
```

Comment:
Does it mean that we only allow empty definitions now? SGTM if this is the only thing we need now. Thanks for removing the definitions that we don't need by now.

Reply:
Yep, that's all I needed to get my integration test to work. Just to have an empty directory mounted into the agent container. 👍